tests/dags/providers/provider_api_scripts/test_smk.py

```diff
@@ -106,6 +114,33 @@ def test_handle_items_data_success():
     assert actual_image_count == 1
 
 
+def test_handle_items_data_success_data():
```

Comment:
Do we need to add the expected filesizes you added to the JSON files to other tests to assert they're being read in correctly? And do you mind adding a test case where filesize is missing to check that it gets set to `None` appropriately?